darshan-util/pydarshan/darshan/experimental/aggregators/mod_agg_iohist.py

```diff
@@ -12,7 +12,7 @@ def mod_agg_iohist(self, mod, mode='append'):
     """
 
     # sanitation and guards
-    supported = ["POSIX", "MPI-IO", "H5D"]
+    supported = ["POSIX", "MPI-IO", "H5D", "PNETCDF_VAR", "PNETCDF_FILE"]
```

Comment:
@tylerjereddy just noticed this when doing a final read-through

I think we can remove `PNETCDF_FILE` from the supported list, as there are no access histogram counters in the file-level records for PnetCDF (they are stored in the variable-level counters). I removed locally and tests pass, so just wanted to double check there wasn't a reason before removing.

Reply:
In the changes to `cli/summary.py` you can see there are similar guards for supported modules for I/O histograms that does not include `PNETCDF_FILE`, which makes total sense, so I just removed it from the supported list here, too. Must have been an oversight.